src/module/actor/data-model-classes/data-model-character-encumbrance-detailed.ts

```diff
@@ -19,6 +19,12 @@ export default class OseDataModelCharacterEncumbranceDetailed
 
   static templateInventoryRow = "";
 
+  static encumbranceSteps = {
```

Comment:
What's your thought process for adding this duplicated static member to the child encumbrance classes?

Reply:
> The base class `encumbranceSteps` seems not usable to me, since it now incorporates all 9 possible steps we have over the encumbrance classes (1/4, 3/8, 1/2, 5/8, 3/4, 7/8, 1/3, 5/9, 7/9).

Huh. News to me. I think that came in from @anthonyronda 's item-based encumbrance changes.

> I don't want to change it in case it gets used elsewhere, plus I think it's more elegant to specify in the class itself what the actual steps are.
>
> I assumed I could overwrite the parent member like this and keep it more readable. Would it be better to rename it to `encumbranceStepsDetailed` or is there a way to actually overwrite the field?

Honestly? This feels like a job for JSDoc and comments. Something like:
```js
export default class OseDataModelCharacterEncumbrance
  implements CharacterEncumbrance
{
  ...
  /**
   * Steps at which encumbrance ought to impact Character move speed.
   * 
   * Built-in encumbrance schemes use: 
   * 
   * - `disabled`: None
   * - `basic`: None; this one uses the `significantTreasure` setting.
   * - `detailed`: `quarter`, `threeEighths`, and `half`
   * - `complete`: `quarter`, `threeEighths`, and `half`
   * - `itembased`: 
   *   - Equipped slots use `oneThird`, `fiveNinths`, and `sevenNinths`
   *   - Packed slots use `fiveEighths`, `threeQuarters`, and `sevenEighths`
   */
  static encumbranceSteps = {
    // Detailed/Complete encumbrance steps
    quarter: 25,
    threeEighths: 37.5,
    half: 50,
    // Item-based encumbrance steps
    fiveEighths: 62.5,
    threeQuarters: 75,
    sevenEighths: 87.5,
    oneThird: 33.34,
    fiveNinths: 55.56,
    sevenNinths: 77.78,
  };
  
  ...
}
```

Reply:
This would definitely clear up the base class a bit, but doesn't really solve the issue of needing only a subset for the derived encumbrances.

Or do I misunderstand something severely?